src/gleam/iterator.gleam

```diff
@@ -431,6 +431,7 @@ fn do_flatten(flattened: fn() -> Action(Iterator(a))) -> Action(a) {
 /// ```gleam
 /// > from_list([[1, 2], [3, 4]])
 /// > |> map(from_list)
+/// > |> from_list
```

Comment:
This is a type error, `from_list` takes a list rather than an iterator.

Reply:
Oops sorry about the error